broker/data-access-layer/iaas/test/iaas.AwsClient.spec.js

```diff
@@ -1,18 +1,11 @@
 'use strict';
 
-const pkgcloud = require('pkgcloud');
-const { CloudProviderClient } = require('@sf/iaas');
+const { AwsClient } = require('@sf/iaas');
 
 describe('iaas', function () {
-  describe('CloudProviderClient', function () {
+  describe('AwsClient', function () {
     this.slow(200);
 
-    const createClientSpy = sinon.spy(pkgcloud.storage, 'createClient');
```

Comment:
Spy was used to test the constructor, added `validateParams` method to check for this.